tests/framework/artifacts.py

```diff
@@ -21,6 +21,7 @@
 )
 from framework.utils import compare_versions, get_kernel_version
 from framework.utils_cpuid import get_instance_type
+import host_tools.network as net_tools
```

Comment:
nit: commit message: maybe ```refactor(tests)``` for better context if ```git log --oneline?```

Reply:
Indeed. This PR already got 2 reviews, so I'll apply this from now on. Thanks.